examples/demo/src/segments/data.ts

```diff
@@ -4,5 +4,9 @@ export default [
     { id: 'ordered_once', name: 'resources.segments.data.ordered_once' },
     { id: 'regular', name: 'resources.segments.data.regular' },
     { id: 'returns', name: 'resources.segments.data.returns' },
-    { id: 'reviewer', name: 'resources.segments.data.reviewer' },
```

Comment:
Please revert this change, and use the simple example to illustrate the functionality. Here, it makes the e-commerce demo worse for no reason.

Reply:
61f53f7